### ISSUE TITLE
Manter entidades de Esfera e Mandato 

### ISSUE THREAD

**Issue by augustohp** [Aeronautics/Spitfire#3]
Existem 2 entidades no banco de dados: **Esfera** e **Mandato** que atualmente não estão sendo preenchidas. O conteúdo delas e dos relacionamentos deve ser intuitivo, qualquer dúvida comentar nesta issue.

Atualmente estamos parseando só um XML de vereaderes, então não existe muita confusão, mas precisamos _urgentementemente_ desses caras populados e por consequência atualizar os `miners` que já estão funcionando.

**Comment by henriquemoody** [comment]
Hoje vou revisar os minners.

**Comment by augustohp** [comment]
Você manja o que é esse esquema de esfera e mandato né!?

**Comment by henriquemoody** [comment]
Sim, um político pode "trafegar" entre as esperas, esse tráfego fica em `mandato`.
Certo?

**Comment by augustohp** [comment]
Isso, acho que é mais correto dizer "pertencer" uma esfera. Mas não acho que esse seja o ideal também =P
Por exemplo: vai existir uma esfera chamada "Câmara Municipal de São Paulo", que irá possuir os vereadores do município e coisa e tal...

Seria bacana também (mais pra frente) criar tabelas de UF e Município, e relacionar com as possíveis esferas. Mas não precisamos disso exatamente agora, eu acho...
